src/plugins/kibana/public/settings/sections/indices/_create.js

```diff
@@ -23,7 +23,7 @@ uiModules.get('apps/settings')
 
   // this and child scopes will write pattern vars here
   const index = $scope.index = {
-    name: 'logstash-*',
+    name: config.get('indexPatternPlaceholder', 'logstash-*'),
```

Comment:
I think you are right `config.get('indexPatternPlaceholder', 'logstash-*')` is redundant because we already have a default value in `src/plugins/kibana/index.js`.
The above example happens if you fill a blank value in the settings page. I think it could be nice to not trigger a query if the index name/pattern is blank. Wdyt ?